addons/l10n_in_check_gst_status/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    "name": "Indian - Check GST Status",
+    "category": "Accounting/Localizations",
```

Comment:
```suggestion
    'countries': ['in'],
    "category": "Accounting/Localizations",
```
Countries must be present inside the l10n modules unless they are not `auto_install`